Makefile

```diff
@@ -33,7 +33,6 @@ APP = gatekeeper
 SRCS-y := main/main.c
 
 # Functional blocks.
-SRCS-y += catcher/main.c
```

Comment:
Checking your branch directly (https://github.com/mengxiang0811/gatekeeper/commits/gk-bp-pull), it's fine. It must be a bug on GitHub, so I'm considering this issue solved.